source/docs/appendices/migration-guides.md

```diff
@@ -51,6 +51,47 @@ This first feature is RMA (Return Merchandize Authorization). In order to add su
   - Your CSS stylesheets should either import the core's `theme/modules/_modules.scss` and `theme/pages/_pages.scss`, or you should ensure that your overrides import
     `theme/modules/User/Order/OrderReturns/OrderReturns`, `theme/modules/User/Return/ReturnsTable/ReturnsTable` and `theme/pages/Account/Orders/Details/ReturnForm/ReturnForm"`,
 
+### Elasticsarch in a dedicated module
+
+Elasticsearch related code has been moved to a dedicated module. As a result, if
+you were using it in 2.4.x you now need to enable the module in your
+`.front-commerce.js`. For a Magento2 based Front-Commerce setup:
+
+```diff
+// .front-commerce.js
+-  modules: [],
++  modules: ["./node_modules/front-commerce/modules/datasource-elasticsearch"],
+   serverModules: [
+     { name: "FrontCommerceCore", path: "server/modules/front-commerce-core" },
+     { name: "Magento2", path: "server/modules/magento2" },
++    {
```

Comment:
we've just discovered with Julien that the module ES module needs to be registered before
the Magento2 one. This needs to be documented properly with a warning here (and then
we can fix that in an upcoming release)